src/main.js

```diff
@@ -53,6 +53,7 @@ const authLimit = rateLimit({
 // But once we have surpassed feature parity, we will instead enable these limits, to help
 // prevent overusage of the api server. With Auth having a lower limit, then non-authed requests.
 
+// TODO: Saving the state key on the database, maybe we don't need this anymore.
```

Comment:
While you are right the initial reason for adding this was for the StateStore, tbh it's actually super helpful in troubleshooting.

Since someone can provide their IP and I can find their logs quickly even if it was days ago. Since we generate so many logs it's nearly useless to find issues without it. So maybe I should just update the comment below to reflect that